src/ontology/mondo-edit.obo

```diff
@@ -451636,7 +451638,7 @@ xref: Orphanet:972 {source="MONDO:equivalentTo"}
 xref: UMLS:C1834559 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:331775"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0002-6601-2165"} ! syndromic disease
 is_a: MONDO:0005336 ! myopathy
-relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="OMIM:160120", source="Orphanet:972"} ! KCNA1
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6218 {source="Orphanet:972"} ! KCNA1
```

Comment:
I don't think the removal of the OMIM source is correct based on https://omim.org/entry/160120. It is possible the data looked differently when pulled from OMIM to create the files used in this PR. 

@sabrinatoro should I add back `source="OMIM:160120"` to this gene association?

Reply:
I agree with your assessment, @twhetzel. We should add back source="OMIM:160120" to this gene association

Reply:
@twhetzel it looks like this change came from the fact that this term is an "omim-included" term. (we do not have plans for these yet)